package.json

```diff
@@ -33,51 +33,51 @@
   },
   "dependencies": {
     "@binance-chain/bsc-connector": "^1.0.0",
-    "@ethersproject/abi": "^5.0.7",
+    "@ethersproject/abi": "^5.1.2",
     "@pancakeswap/uikit": "^0.32.2",
     "@reduxjs/toolkit": "^1.5.0",
-    "@types/react": "^17.0.2",
-    "@types/react-dom": "^17.0.1",
+    "@types/react": "^17.0.5",
+    "@types/react-dom": "^17.0.4",
     "@types/react-router-dom": "^5.1.5",
     "@types/styled-components": "^5.1.7",
     "@web3-react/core": "^6.1.9",
     "@web3-react/injected-connector": "^6.0.7",
-    "@web3-react/walletconnect-connector": "^6.1.9",
+    "@web3-react/walletconnect-connector": "^6.2.0",
     "bignumber.js": "^9.0.0",
     "canvas-confetti": "^1.3.3",
-    "chart.js": "^2.9.4",
-    "date-fns": "^2.17.0",
-    "ethers": "^5.0.31",
+    "chart.js": "2.9.4",
```

Comment:
I kept chart.js at 2.9.4 because it messed with the lottery chart. I would like to revisit our "charting strategy" once the new lottery work commences.

@ChefHutch